esp/esp/program/modules/handlers/commmodule.py

```diff
@@ -109,6 +110,11 @@ def commprev(self, request, tl, one, two, module, extra, prog):
         esp_firstuser = ESPUser(firstuser)
         contextdict = {'user'   : ActionHandler(esp_firstuser, esp_firstuser),
                        'program': ActionHandler(self.program, esp_firstuser) }
+
+        htmlbody = unicode(loader.get_template('email/default_email_html.txt').render(DjangoContext({'msgbdy': body,
```

Comment:
I think the more standard thing is to use `template.render_to_string()` rather than `unicode()`, and in that case I don't think you need to call the `Context` instance yourself, you just pass a dict.